setup.py

```diff
@@ -111,6 +111,11 @@ def run(self):
     # test_suite = 'nose.collector',
     # test_requires = ['nosetest'],
 
+    tests_require=[
+        # used to provide unit test capabilities
+        'mock >= 2.0.0',
```

Comment:
Current policy is to add additional dependencies only in case of uttermost necessity. All provided tests can be expressed via standard youtube-dl's `_TESTS` mechanism without any problem.